### ISSUE TITLE
[Question] Auto import module.

### ISSUE THREAD

**Issue by marcem7D0** [KronicDeth/intellij-elixir#3663]
I used VSCode with the ElixirLS plugin where if you type a module it will suggest to import it. I did not see this or at least I couldn't find a way to auto import module alias, as result I had to type it manually. Is it possible with this plugin in Intellij Idea?

**Comment by sh41** [comment]
No, not today — apologies for the long delay in answering. The plugin has no auto-import action for aliases, so the `alias` line still has to be typed by hand.

What does exist, as of [24.0.0](https://github.com/KronicDeth/intellij-elixir/releases/tag/24.0.0), is the `UnresolvableModuleQualifier` inspection (shipped in [#3821](https://github.com/KronicDeth/intellij-elixir/pull/3821)). It is enabled by default and flags a module qualifier that isn't defined or aliased in the current scope as an error. So the IDE already recognises exactly the situation you describe — what's missing is only the fix action that would insert the `alias` for you and shorten the reference, the way ElixirLS suggests it.

That fix action is on the backlog: scoped, wanted, not yet built. I can't give you a date, but it isn't something that was considered and rejected.

This issue was looked at as part of an AI-assisted review of the backlog, and a human checked this reply before it was posted. If any of it is wrong, please say so here.
